src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackSectionBlock.php

```diff
@@ -44,6 +44,9 @@ public function field(string $text, bool $markdown = true): self
             'text' => $text,
         ];
 
+        // Maximum number of items is 10
+        $this->options['fields'] = \array_slice($this->options['fields'], 0, 10);
```

Comment:
Instead of silently removing items, I would throw an exception.

Reply:
Yeap, you're right! ( seems this doesn't pop up on my notifications :-( )
I'll take care of this right away!

Reply:
@fabpot, agree with exception message?